src/main/java/com/linkedin/kmf/services/JolokiaService.java

```diff
@@ -36,14 +36,14 @@ public JolokiaService(Map<String, Object> props, String name) throws Exception {
   public synchronized void start() {
     if (_isRunning.compareAndSet(false, true)) {
       _jolokiaServer.start();
-      LOG.info(_name + "/JolokiaService started at port 8778");
+      LOG.info("{}/JolokiaService started at port 8778", _name);
```

Comment:
Yes it is hardcoded as 8778.. It is intended for demo but not PROD deployment. So it is probably OK.